bot/exts/filters/filtering.py

```diff
@@ -6,13 +6,12 @@
 
 import arrow
 import dateutil.parser
-import discord.errors
```

Comment:
what exactly is the reason for this?

Reply:
As mentioned [here](https://github.com/python-discord/bot/pull/2069#pullrequestreview-869807491), swapping from this usage of `discord.errors` namespace to exact imports would keep the consistency of how the `discord` package is used across this file.